### PR TITLE
Consolidate SQL db access

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph 

:blowfish: 

cc @biyeun @calellowitz

### CODE REVIEW

File: corehq/form_processor/interfaces/processor.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from couchdbkit.exceptions import BulkSaveError
 from redis.exceptions import RedisError
 
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors, FormAccessors
```
Comment (dimagimon) [inline corehq/form_processor/interfaces/processor.py]: F401 'CaseAccessors' imported but unused

Comment (dimagimon) [inline corehq/form_processor/interfaces/processor.py]: F401 'FormAccessors' imported but unused

File: corehq/form_processor/backends/sql/dbaccessors.py [hunk 2]
```diff
@@ -1,5 +1,14 @@
+from datetime import datetime
+from django.db import transaction
+from django.db.models import Prefetch
 from corehq.form_processor.models import XFormInstanceSQL, CommCareCaseSQL
 
+from corehq.form_processor.exceptions import XFormNotFound, CaseNotFound
+from corehq.form_processor.interfaces.dbaccessors import AbstractCaseAccessor, AbstractFormAccessor
+from corehq.form_processor.models import (
```
Comment (dimagimon) [inline corehq/form_processor/backends/sql/dbaccessors.py]: F811 redefinition of unused 'CommCareCaseSQL' from line 4

Comment (dimagimon) [inline corehq/form_processor/backends/sql/dbaccessors.py]: F811 redefinition of unused 'XFormInstanceSQL' from line 4

**Comment by benrudolph** [comment]
:+1: once tests pass

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
self merging

**Pull request merged**